config/crd/full/serving.kserve.io_inferenceservices.yaml

```diff
@@ -10461,8 +10494,41 @@ spec:
                         - concurrency
                         - rps
                       type: string
+                    scaleMetricType:
+                      type: string
                     scaleTarget:
                       type: integer
+                    scaler:
+                      properties:
+                        maxReplicas:
+                          type: integer
+                        metricBackend:
+                          enum:
+                            - prometheus
+                            - graphite
+                          type: string
+                        metricQuery:
+                          type: string
+                        minReplicas:
+                          type: integer
+                        queryParameters:
+                          type: string
+                        queryTime:
+                          type: string
+                        scaleMetric:
+                          enum:
+                            - cpu
```

Comment:
Should we support custom metrics or is this just for cpu an mem for now and later we will add the former?